frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/_add-capacity-modal.scss

```diff
@@ -1,37 +1,31 @@
 @import '~@patternfly/patternfly/sass-utilities/colors';
 
-.add-capacity-modal__hide {
-  display: none;
+.ceph-add-capacity__current-capacity {
+  display: flex;
 }
 
-.add-capacity-modal__input--width {
-  max-width: 250px !important;
+.ceph-add-capacity__current-capacity--loading {
+  width: 7.9rem;
+  margin-top: 0.2rem;
 }
 
-.add-capacity-modal__input .dropdown {
-  margin-left: var(--pf-global--spacer--md);
+.ceph-add-capacity__current-capacity--text {
+  margin-right: 0.5rem;
 }
 
-.add-capacity-modal__help-text-error {
-  color: $pf-color-red-100;  
+.ceph-add-capacity__form {
+  display: flex;
+  align-items: center;
 }
 
-.add-capacity-modal__help-text {
-  margin-top: -15px;
+.ceph-add-capacity__input {
+  max-width: 250px !important;
```

Comment:
`!important` should not be required? Could you please explain where the style clash is happening?

Reply:
Yes, this was present before hand, hence did not touch.
Will look into it, and do whats needful.

Reply:
yes, some generic css is having `max-width: 100px` for all input sizes of number type.
Hence `!important` is used to override that behavior